src/ia-dropdown.ts

```diff
@@ -22,6 +22,26 @@ export class IaDropdown extends LitElement {
 
   @property({ type: Boolean, attribute: true }) displayCaret = false;
 
+  /**
+   * To allow customizing whether the dropdown menu automatically closes
+   * when an option is selected.
+   */
+  @property({ type: Boolean, attribute: true }) closeOnSelect = false;
+
+  /**
+   * To allow customizable behavior where clicking on the caret or
+   * the main button itself should be handled differently.
+   */
+  @property({ type: Boolean, attribute: true }) openViaButton = true;
+
+  @property({ type: Boolean, attribute: true }) openViaCaret = true;
+
+  /**
+   * To allow customizing whether the currently selected option should still
+   * be shown in the dropdown. By default, it is not.
+   */
+  @property({ type: Boolean, attribute: true }) includeSelectedOption = false;
```

Comment:
q: can we set this default to current component behavior, too? l

https://archive.org/details/cd_taylor-swift_taylor-swift - shrink to mobile view - right now selected option is not included.. would like to keep those current behaviors as defaults if possible...

Reply:
Agreed -- I think `false` is the correct default for current behavior then (_don't_ include the selected option)?

Reply:
New comment should hopefully be clearer about this.